### ISSUE TITLE
Default padding at .wp-block-button

### ISSUE THREAD

**Issue by JohnPixle** [Codeinwp/otter-blocks#204]
There appears to be an issue with the padding of the buttons in the Gutenberg editor. The` .wp-block-button` class seems it has a padding of 0.75em and it applies on the frontend, at the core _button blocks_ as well as the Otter _buttons group_. 

This causes an alignment issue with the rest of the blocks because the button(s) appear with an inset that is due to the padding.

In the editor however, things look normal:
![Screenshot 2020-05-22 at 20 12 12](https://user-images.githubusercontent.com/4193672/82692700-d9a19a80-9c68-11ea-9e30-5847f467a2dd.png)

The issue happens on the frontend:
![Screenshot 2020-05-22 at 20 12 19](https://user-images.githubusercontent.com/4193672/82692930-3d2bc800-9c69-11ea-8289-d6cc37958fe1.png)

Inspecting the element you see the padding applied on the ` .wp-block-button` class which is in effect.

![Screenshot 2020-05-22 at 20 15 22](https://user-images.githubusercontent.com/4193672/82692982-546ab580-9c69-11ea-82d2-4fc70bfd9aaf.png)

I tested at a local site, without Otter disabled, and the buttons work fine (they do not have this padding).

I spotted this issue at other starter sites as well ( examples are [here](https://demosites.io/scholar-gb/courses/) and [here](https://demosites.io/lawyers-gb/)) :

![Screenshot 2020-05-22 at 20 20 44](https://user-images.githubusercontent.com/4193672/82693529-63059c80-9c6a-11ea-8a4a-7394e0850ca3.png)
![Screenshot 2020-05-22 at 20 24 56](https://user-images.githubusercontent.com/4193672/82693531-64cf6000-9c6a-11ea-8170-fde39ccbc928.png)

Let me know your thoughts.

TLDR: Here is a quick video explanation of the issue
https://www.loom.com/share/1a155d6b06c64772a882b7650908641b

cc @selul

**Comment by selul** [comment]
Hi @JohnPixle, this is from Neve, will be fixing it in the upcoming release
